src/unit_tests/wazuh_db/test_wdb_parser.c

```diff
@@ -518,56 +532,56 @@ void test_integrity_clear_ok(void **state)
-        // cmocka_unit_test_setup_teardown(test_update_info_error, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_update_info_ok, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_clean_old_entries_error, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_clean_old_entries_ok, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_scan_info_update_noarg, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_scan_info_update_error, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_scan_info_update_ok, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_scan_info_fim_check_control_error, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_scan_info_fim_check_control_ok, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_syscheck_load_error, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_syscheck_load_ok, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_fim_delete_error, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_fim_delete_ok, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_syscheck_save_noarg, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_syscheck_save_invalid_type, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_syscheck_save_file_type_error, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_syscheck_save_file_nospace, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_syscheck_save_file_type_ok, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_syscheck_save_registry_type_error, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_syscheck_save_registry_type_ok, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_syscheck_save2_error, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_syscheck_save2_ok, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_integrity_check_error, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_integrity_check_no_data, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_integrity_check_checksum_fail, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_integrity_check_ok, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_integrity_clear_error, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_integrity_clear_ok, test_setup, test_teardown),
-        // cmocka_unit_test_setup_teardown(test_invalid_command, test_setup, test_teardown)
+
+int main() {
+    const struct CMUnitTest tests[] = {
```

Comment:
Try to separate in different blocks of code for each function

Reply:
All tests in _test_wdb_parser_ test the same function, `wdb_parse_syscheck` so they cannot be separated by function.